src/box/box.cc

```diff
@@ -299,7 +299,7 @@ box_storage_init(void);
 /**
  * Broadcast the current instance status
  */
-static void
+void
 box_broadcast_status(void);
```

Comment:
You can remove this now, as you have the declaration in the header.

Reply:
Indeed, removed